tests/test_models.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
do we need this file?

Reply:
yes, if there isn't a single test CI will fail